addons/spreadsheet/models/res_currency.py

```diff
@@ -12,7 +12,7 @@ def get_company_currency_for_spreadsheet(self, company_id=None):
         This function is meant to be called by the spreadsheet js lib,
         hence the formatting of the result.
 
-        :company_id int: Id of the company
+        :param int company_id: Id of the company
         :return: dict of the form `{ "code": str, "symbol": str, "decimalPlaces": int, "position":str }`
```

Comment:
```suggestion
        :return: has the form ``{ "code": str, "symbol": str, "decimalPlaces": int, "position":str }``
        :rtype: dict
```

`Must` have the form?